### DISCUSSION TITLE
.NET MAUI Image Caching

### DISCUSSION THREAD

**Discussion by albilaga** [dotnet/maui#25902] [General] [answered]
I am not sure actually if this should be discussion or bug but want to ask according to [this line of code](https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/UriImageSource.cs#L96) URIImageSource actually not enabling any caching at all even though `CachingEnabled` is `true`? Is this my understanding correct? If yes, so the [docs in here](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/image?view=net-maui-9.0#image-caching) should not mention any image caching at all right?

**Comment by tonyedwardspz** [comment, marked as answer]
There's a related thread to this in #9138.
